guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -21,6 +21,14 @@ For more information about configuring a web server, see {RHELDocsBaseURL}9/html
 endif::[]
 
 .Procedure
+ifdef::katello[]
+. On your server, enable the required repositories:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf config-manager --add-repo=https://yum.theforeman.org/pulpcore/nightly/el9/x86_64/
```

Comment:
```suggestion
# dnf config-manager --add-repo=https://yum.theforeman.org/pulpcore/{PulpcoreVersion}/el9/x86_64/
```

I like that we fix the procedure for upstream Foreman+Katello, but I currently see two minor issues:

* We should use the attribute that tracks the Pulpcore version. Another issue: the attribute should be "nightly" on the "master" branch in foreman-documentation -> we can fix this in a follow-up PR.
* To my knowledge, "dnf config-manager" cannot set a URL for the GPG key. -> we can convert this to a simpler "cat EOF" thingy (https://linuxize.com/post/bash-heredoc/#basic-heredoc-examples) in a follow-up PR.

Note that both issues need a special "if master" then "no GPG key and then no attribute", but we can use the `DocState` attribute for make that happen. I can provide a follow-up PR for that.